webapp/src/components/VerticalTabs.tsx

```diff
@@ -53,6 +54,19 @@ function VerticalTabs(props: VerticalTabProps) {
   const filteredTabs = tabs.filter((tab) => evaluateFeatureFlags(tab.tag))
   const defaultTabKey = filteredTabs[defaultTabIndex ?? 0]?.path
 
+  const getIcon = (tabName: string) => {
```

Comment:
We should pass the icon through the tabs object (VerticalTabItem) instead of hardcoding them here. This component is intended to be generic

Reply:
Updated to pass the icon through VerticalTabItem instead of hardcoding